cmd/nerdctl/image_save_test.go

```diff
@@ -25,7 +25,10 @@ import (
 )
 
 func TestSaveById(t *testing.T) {
-	base := testutil.NewBase(t)
+	base := testutil.NewBaseWithNamespace(t, testutil.Identifier(t))
```

Comment:
Could you add a comment about namespacing?

Reply:
Done and pushed.